rtl/src/errors.rs

```diff
@@ -18,6 +18,9 @@ pub enum Error {
     GCore(#[from] GCoreError),
     #[error("codec: {0}")]
     Codec(#[from] CodecError),
+    #[cfg(not(target_arch = "wasm32"))]
+    #[error("codec: {0}")]
+    GClient(#[from] gclient::Error),
```

Comment:
-> GSdk?

Reply:
` gclient::Error` is more obvious, and I don't like aliases)